modules/default-requirements-central-services.adoc

```diff
@@ -32,7 +32,7 @@ recommend using a hostPath volume.
 ====
 * Use Solid-State Drives (SSD) for best performance.
 However, you can use another storage type if you do not have SSDs available.
-* If you use a web proxy or firewall, you must configure bypass rules to allow traffic for the `definitions.stackrox.io` domain and enable {product-title} to trust your web proxy or firewall. Otherwise, updates for vulnerability definitions will fail.
+* If you use a web proxy or firewall, you must configure bypass rules to allow traffic for the `definitions.stackrox.io` domain and enable {product-title-short} to trust your web proxy or firewall. Otherwise, updates for vulnerability definitions will fail. You must also ensure that Central can communicate with secured clusters on HTTPS port 443.
```

Comment:
I think that last sentence is only necessary if Central were to initiate the connections with the secured clusters, but it's the other way around. So, I don't think this sentence is true

Reply:
Will moving this sentence to the secured clusters section address the issue? There wasn't anything about opening the firewall for port 443 in the secured clusters section so that's why I had it under Central since there is the firewall mentioned.

Reply:
It was likely documented at central because there is an _incoming_ connection from Sensor to Central on port 443.
Therefore the firewall securing central needs to allow incoming traffic to central on port 443.
I'd tend towards leaving this information here, but rephrasing it, e.g.
"You must also ensure that incoming connections from Sensors/Secured Clusters to Central on port 443 are possible"